contracts/ERC734.sol

```diff
@@ -1,55 +1,37 @@
-pragma solidity ^0.6.2;
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.6.9;
+
+import "./Storage/Storage.sol";
+import "./Library/LibraryLock.sol";
+import "./Interface/IERC734.sol";
 
-import "./IERC734.sol";
 
 /**
  * @dev Implementation of the `IERC734` "KeyHolder" interface.
  */
-contract ERC734 is IERC734 {
-    uint256 public constant MANAGEMENT_KEY = 1;
-    uint256 public constant ACTION_KEY = 2;
-    uint256 public constant CLAIM_SIGNER_KEY = 3;
-    uint256 public constant ENCRYPTION_KEY = 4;
-
-    bool private identitySettled = false;
-    uint256 private executionNonce;
-
-    struct Execution {
-        address to;
-        uint256 value;
-        bytes data;
-        bool approved;
-        bool executed;
-    }
-
-    mapping (bytes32 => Key) private keys;
-    mapping (uint256 => bytes32[]) private keysByPurpose;
-    mapping (uint256 => Execution) private executions;
-
+contract ERC734 is Storage, LibraryLock, IERC734 {
```

Comment:
This management of storage is weird.
Here, ERC734, because it inherits Storage, also has all storage of ERC735 with claims, it doesn't make sense.

Reply:
if you have a better suggestion for this, go ahead

Reply:
i agree, you need to refactor in a way that you inherit `storage` only once, otherwise it could create issues, the solution would be to put ERC734 and ERC735 in a single contract inheriting from IERC734, IERC735 and storage

Reply:
`storage` is currently inherited only once. 
What kind of issuescould this cause?

Reply:
ah ok sorry, didn't check well, there is no ERC735 contract it is Identity and ClaimIssuer, for me it's ok like this

Reply:
I think we can do something based on interfaces, getters and setters, i'll try!

Reply:
Ok, go ahead !👍